modules/modal/react/lib/ModalContent.tsx

```diff
@@ -215,6 +220,8 @@ const ModalContent = ({
         handleClose={handleClose}
         padding={padding}
         transformOrigin={transformOrigin}
+        aria-modal={true}
```

Comment:
It looks like `aria-modal=true` is supposed to replace the functionality of setting `aria-hidden=true` on children, but isn't fully supported yet.

Do you know if this technique will work with multiple modal dialogs stacked? The `aria-hidden` should work in that case, but I'm not sure if there are multiple `aria-modal=true`. Should the previous modal get a set back to "false" or be removed?

Reply:
I created a test: https://codesandbox.io/s/stacked-modal-qhcp6

Reply:
@lychyi Thanks for taking a look!:)
@NicholasBoll good question! I will ask Peter. I would assume since we have aria-hidden for the first popup aria-modal would be ignored.

Reply:
Maybe that will work as long as we apply `aria-hidden` to things. I assume broad support for `aria-modal` is a ways off, so we can worry about it then.